suite-native/module-onboarding/src/screens/AnalyticsConsentScreen.tsx

```diff
@@ -65,8 +65,12 @@ export const AnalyticsConsentScreen = () => {
                 </Stack>
             </Card>
             <Stack spacing="small" style={applyStyle(buttonsWrapperStyle)}>
-                <Button onPress={handleRedirect}>Not now</Button>
-                <Button onPress={handleAnalyticsConsent}>Allow anonymous data collection</Button>
+                <Button data-testID="allow-userData" onPress={handleAnalyticsConsent}>
```

Comment:
Switched button order, "Not now" should be first.

Reply:
`Not now` button is now first.